tools/xcms_xcmsset/abims_xcms_xcmsSet.xml

```diff
@@ -30,6 +30,7 @@
 
         #if $methods.method == "CentWave":
             @COMMAND_CENTWAVE@
+			peaklist $methods.CentWaveAdv.peaklist
```

Comment:
There is an indentation issue here.
Can you use 4 spaces instead of tabulation?